### DISCUSSION TITLE
VSCodeWorkbench mount failed with "Failed to connect to relay url Code: 1006 Reason: "

### DISCUSSION THREAD

**Discussion by shawnshawnlian** [community/community#28597] [Codespaces]
Whenever I try going into my codespaces, it says "VSCodeWorkbench mount failed with "Failed to connect to relay url Code: 1006 Reason: ". Flask doesn't work if you go to the visual studio application, so I need to get into code.cs50.io to complete my project.

**Comment by gcsiapn** [comment]
same problem.

**Comment by srepollock** [comment]
I'm now running into this issue. Is there any ideas with a fix?

**Comment by helloworld3200** [comment]
I'm also having this problem.

**Comment by alexsharris** [comment]
Also having this problem[
<img width="906" alt="Screenshot 2024-08-14 at 1 12 42 PM" src="https://github.com/user-attachments/assets/31d6c84e-7844-469c-9c30-90d2ec213131">
](url)
